index.html

```diff
@@ -62,7 +62,7 @@
 
   <header class="text-center title"><b>CALCULATOR</b></header>
     
-   <center><p class="paragraph">Hi, I am Yoshitha Rathnayake. This is the Calculator App that I made using Python Programming Language. You can download it by clicking the below link. It's saying there are viruses, but don't worry they are lies.</p></center>
+   <center><p class="paragraph">Hi, I am Yoshitha Rathnayake. This is the Calculator App.This app is made by me using the Python Programming Language. You can download it by clicking on the link given below. This app is free from any kind of virus and you can use it safely.</p></center>
```

Comment:
Can you made a new issue for this and do so it will considered as a new commit for you @Abhishekgupta204